backend/fbdev.c

```diff
@@ -111,10 +140,17 @@ static bool twin_fbdev_apply_config(twin_fbdev_t *tx)
         return false;
     }
 
-    /* Check bits per pixel */
-    if (tx->fb_var.bits_per_pixel != 32) {
-        log_error("Failed to set framebuffer bpp to 32");
-        return false;
+    /* Examine the framebuffer format */
+    if (tx->fb_var.bits_per_pixel == 16) {
```

Comment:
Improve the checks.

Reply:
> Improve the checks.

I need more hints.

Reply:
See [Incorrect calculation of color data size using bits_per_pixel](https://github.com/espressif/esp-idf/issues/9729) and the corresponding commit.
`bits_per_pixel` can be 16, 24, and 32.